tools/dotnet-msidentity/Program.cs

```diff
@@ -209,25 +235,41 @@ private static async Task<int> HandleUpdateProject(ProvisioningToolOptions provi
                 IsRequired = false
             };
 
+        private static Option EnableIdTokenOption() =>
```

Comment:
This is leaking things of a different level of abstraction.
Please explain the use case.

Reply:
Will be removed in an upcoming PR.